src/App.tsx

```diff
@@ -12,161 +21,48 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={
+              ({ isActive }) => cn(
+                'navbar-item', {
+                  'has-background-grey-lighter': isActive,
+                },
+              )
+            }
+            to="/"
+          >
             Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          </NavLink>
+
+          <NavLink
+            className={
+              ({ isActive }) => cn(
+                'navbar-item', {
+                  'has-background-grey-lighter': isActive,
+                },
+              )
+            }
+            to="/people"
           >
             People
-          </a>
+          </NavLink>
```

Comment:
1. Make a separate component for navigation
2. You can also render NavLinks as a list